apps/igloo-kit-cli/src/framework.rs

```diff
@@ -15,15 +15,15 @@ pub enum Insights {
 pub enum TopLevelObjects {
     Ingestion,
     Flow,
-    Dataframe,
+    DataModel,
```

Comment:
The dictionary is happy with `Dataframe` as one single word, but not `Datamodel`. I don't feel strongly either way.

Reply:
I don't feel strrongly either way either. I think this is fine.